braindecode/models/eegsimpleconv.py

```diff
@@ -13,8 +13,8 @@
 
 import torch
 from torch import nn
+from torchaudio.transforms import Resample
 
-from braindecode.models.modules import Resample
```

Comment:
Basically, I include torch audio as a dependency for braindecode. Until now, for some functions, we had to copy the code from torch audio to avoid dependency.

With the filterbank module, we would have to add a good part of the filter module, so I decided to directly include torch audio and get rid of all the copied code.

I'll improve the PR description by including this detail.